apps/condo/domains/acquiring/schema/MultiPayment.test.js

```diff
@@ -143,11 +144,11 @@ describe('MultiPayment', () => {
 
                     const integrationClient = await makeClientWithNewRegisteredAndLoggedInUser()
                     await createTestAcquiringIntegrationAccessRight(admin, acquiringIntegration, integrationClient.user)
-                    const [_, updatedMultiPaymentAttrs] = await updateTestMultiPayment(integrationClient, multiPayment.id, {
+                    let [updatedMultiPayment] = await updateTestMultiPayment(integrationClient, multiPayment.id, {
```

Comment:
please use `let`only if you need reassign value for a variable, in all other cases please use `const`